core/src/scss/components/pull-quote/_pull-quote.scss

```diff
@@ -0,0 +1,18 @@
+@charset "UTF-8";
+
+//
+// Pull-quote
+//
+// Pull-quote styling.
+//
+//
+.su-pull-quote-left,
+aside {
```

Comment:
Hi @cjwest  Looks like this would apply the mixin to all `aside` elements so perhaps using a class approach is safer here. I also left a message on Slack, any chance we could rename this from pull-quote to something a little more general. The name "pull-quote" did make me expect an actual quote before seeing it 😃